docs/docs/modeling.md

```diff
@@ -114,6 +114,25 @@ classDiagram
 
 ## AssignmentSheet - 分担票
 
+Tenant's task assignment table.  
+The assignmentSheet is including startDate,endDate and TenantsWork.
```

Comment:
I think it's better to say this `AssignmentSheet` cannot exist without `StartDate`, `EndDate` and `TenantsWork` here.

Reply:
Thanks for the advice. I have added the constraint.